src/index.html

```diff
@@ -11,6 +11,59 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+      >
+
+        <div class="search__bar">
+          <img
+          class="search__img__big"
+          src="images/Search.svg"
+          alt="Search-image"
+          >
+
+          <input
+          data-qa="keypress"
+          class="search__input"
+          type="search"
+          name="bigSearchBar"
+          placeholder="Try &quot;Los Angeles&quot;"
+          id="big_search_bar"
+          >
+        </div>
+
+      </form>
```

Comment:
check code style formatting for the whole file
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

```suggestion
    <main class="main">
      <form
        action="#"
        method="get"
        class="form"
        data-qa="big"
      >
        <div class="search__bar">
          <img
            class="search__img__big"
            src="images/Search.svg"
            alt="Search-image"
          >

          <input
            data-qa="keypress"
            class="search__input"
            type="search"
            name="bigSearchBar"
            placeholder="Try &quot;Los Angeles&quot;"
            id="big_search_bar"
          >
        </div>
      </form>
```